cmd/lncli/commands.go

```diff
@@ -2656,6 +2658,21 @@ func updateChannelPolicy(ctx *cli.Context) error {
 	return nil
 }
 
+var fishCompletionCommand = cli.Command{
+	Name:   "fish-completion",
+	Hidden: true,
+	Action: func(c *cli.Context) error {
+		completion, err := c.App.ToFishCompletion()
+		if err != nil {
+			return err
+		}
+		// we don't want to suggest files, so we add this first line to the completions
+		completion = fmt.Sprintf("complete -c %q -f \n", c.App.Name) + completion
```

Comment:
Could be a one liner: `fmt.Printf("complete -c %q -f \n%v", c.App.Name, completion)`.